src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
+const azurePortalUrl = 'https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    if (await isDeployToAzureExtensionInstalled()) {
+        await getClusterAndExecute(target, configurePipelineCommand);
+    }
+}
+
+export async function browsePipeline(context: IActionContext, target: any): Promise<void> {
+    const deploymentCenterUrl: string = await getDeploymentCenterUrl(target);
```

Comment:
💡 `const` level annotations is not required as the `funct` will return the desired type.